cms/static/js/views/group_configuration_details.js

```diff
@@ -36,8 +37,9 @@ function(BaseView, _, gettext, str) {
             var attrs = $.extend({}, this.model.attributes, {
                 groupsCountMessage: this.getGroupsCountTitle(),
                 usageCountMessage: this.getUsageCountTitle(),
-                outlineAnchorMessage: this.getOutlineAnchorMessage(),
-                index: this.model.collection.indexOf(this.model)
+                courseOutlineUrl: this.model.collection.outlineUrl,
+                index: this.model.collection.indexOf(this.model),
+                HtmlUtils: HtmlUtils
             });
 
             this.$el.html(this.template(attrs));
```

Comment:
Since you're in here, you might want to change this to use ``HtmlUtils.setHtml`` etc.